### PR TITLE
Create autoupdate.yaml

### PR DESCRIPTION
**Pull request by Vasilije1990**
## Summary by CodeRabbit

- **New Features**
	- Introduced automated dependency updates for `pip`, `github-actions`, and `docker` ecosystems.
	- Added a workflow to send automated greetings and requests for information on pull requests and issues.

- **Chores**
	- Configured Dependabot settings for managing dependencies efficiently.

### CODE REVIEW

File: .github/workflows/autoupdate.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Configuration: https://dependabot.com/docs/config-file/
+# Docs: https://docs.github.com/en/github/administering-a-repository/keeping-your-dependencies-updated-automatically
+
+
+version: 2
+
```
[the remark this answers is not in the document]
Comment (dexters1) [reply .github/workflows/autoupdate.yaml:6]: I think this comment is correct it needs to be named dependabot.yml and located in the .github folder

File: .github/dependabot.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Configuration: https://dependabot.com/docs/config-file/
+# Docs: https://docs.github.com/en/github/administering-a-repository/keeping-your-dependencies-updated-automatically
+
+version: 2
+
+updates:
+  - package-ecosystem: "pip"
```
Comment (dexters1) [inline .github/dependabot.yaml:7]: We can also have it check the npm package ecosystem for the cognee frontend

Comment (Vasilije1990) [reply .github/dependabot.yaml:7]: We can, but Boris is not maintaining it. So, for now should be paused I think

Comment (Vasilije1990) [reply .github/dependabot.yaml:7]: @coderabbitai review

**Comment by Vasilije1990** [comment]
> Warning
> 
> ## Rate limit exceeded
> @Vasilije1990 has exceeded the limit for the number of commits or files that can be reviewed per hour. Please wait **0 minutes and 6 seconds** before requesting another review.
> 
> ⌛ How to resolve this issue?
> 🚦 How do rate limits work?
> 📥 Commits
> 📒 Files selected for processing (2)
> ## Walkthrough
> The changes introduce a Dependabot configuration file in `.github/workflows/autoupdate.yaml` to automate dependency updates for various ecosystems, including `pip`, `github-actions`, and `docker`. The configuration specifies version 2 of Dependabot, sets a weekly update schedule, and allows a maximum of 50 open pull requests per ecosystem. Specific rules for handling the "poetry" dependency are included, allowing for both automatic updates and manual management.
> 
> ## Changes
> File Path	Change Summary
> .github/workflows/autoupdate.yaml	Added Dependabot configuration for `pip`, `github-actions`, and `docker` ecosystems with specific update strategies and rules for "poetry".
> ## Sequence Diagram(s)
> ```mermaid
> sequenceDiagram
>     participant User
>     participant Dependabot
>     participant Ecosystem
> 
>     User->>Dependabot: Request dependency updates
>     Dependabot->>Ecosystem: Check for updates weekly
>     Ecosystem-->>Dependabot: Return available updates
>     Dependabot->>User: Create pull request for updates
> ```
> > 🐰 "In the garden where updates bloom,
> > Dependabot dances, clearing the gloom.
> > With `pip` and `docker`, it hops with delight,
> > Keeping our packages shiny and bright!
> > 'Poetry' waits, a special case,
> > Managed with care, in its own little space." 🌼
> 
> Thank you for using CodeRabbit. We offer it for free to the OSS community and would appreciate your support in helping us grow. If you find it useful, would you consider giving us a shout-out on your favorite social media?
> 
> ❤️ Share
> 🪧 Tips

@coderabbitai review

**Pull request merged**